projects/kit/src/lib/masks/time/utils/stringify-time.ts

```diff
@@ -18,6 +18,7 @@ export function maskitoStringifyTime(
         ...timeSegmentMaxValues,
     };
 
+    const hasMeridiem = mode?.includes('AA');
```

Comment:
Is optional chaining really required ?

https://github.com/taiga-family/maskito/blob/2d8c40b6ab728b4e58c2e4d08ae56b8ec8f33f21/projects/kit/src/lib/masks/time/time-params.ts#L3-L4

Reply:
```suggestion
    const hasMeridiem = mode.includes('AA');
```